tests/nextflow.config

```diff
@@ -1,5 +1,6 @@
-/*
-========================================================================================
-    Nextflow config file for running tests
-========================================================================================
-*/
+includeConfig 'https://raw.githubusercontent.com/nf-core/modules/0094ae45ba8f5a2e30644d4e252970775a03ee91/tests/config/test_data.config'
```

Comment:
I don't think there's any need to point to a specific commit for this file, we mostly touch it to add files to it

Reply:
Yeah, I think the commit makes sense in this case given we would ideally like to pin the test data used in nf-test. Might be something we need to consider more widely on nf-core because this could break tests.